# Generated by Django 5.1.2 on 2025-01-27 07:24

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('jobsapp', '0015_employeeprofile_delete_internalemployee'),
    ]

    operations = [
        migrations.CreateModel(
            name='ImportError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('timestamp', models.DateTimeField(auto_now_add=True)),
                ('row_number', models.IntegerField()),
                ('employee_name', models.CharField(max_length=255)),
                ('file_no', models.CharField(blank=True, max_length=100, null=True)),
                ('error_type', models.CharField(max_length=100)),
                ('error_message', models.TextField()),
                ('raw_values', models.JSONField()),
            ],
            options={
                'verbose_name': 'Import Error Log',
                'verbose_name_plural': 'Import Error Logs',
                'ordering': ['-timestamp'],
            },
        ),
    ]
